### ISSUE TITLE
UI: navgation occludes top content on narrow window

### ISSUE THREAD

**Issue by Marwe** [nsupdate-info/nsupdate.info#315]
The navigation bar can occlude content, if the window is too narrow (here: half screen = 800 px), "add host" is covered then, see screenshot.

![screenshot: occlusion](https://cloud.githubusercontent.com/assets/2165004/26664832/0b54502c-4695-11e7-94e4-a1354d0f4b35.png)

**Comment by ThomasWaldmann** [comment]
Yeah, the point where it switches to the narrow-screen display (with burger menu) seems to be "too late".

**Comment by ThomasWaldmann** [comment]
It's not as bad as in the screenshot any more.

It still is slightly ugly overlapping the "Your Hosts [Add Host]" heading.

**Comment by ThomasWaldmann** [comment]
Tried to reproduce, there is no overlap anymore.

Maybe that was fixed when bootstrap was upgraded to latest version.
